packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -1430,6 +1444,8 @@ export const DialogueMatchingPage = ({classes}: {
 
   const updateCurrentUser = useUpdateCurrentUser()
   const currentUser = useCurrentUser();
+  const { location, query, params } = useLocation()
```

Comment:
Is this being used for anything?

Reply:
Oh, I see it's being used for the `dialogueCheckId` query.  Are the other two things used anywhere?

Reply:
nope, removing